src/content_contact.js

```diff
@@ -0,0 +1,33 @@
+import { resetContentWrapper } from './tabbed_navigation';
+
+export default function displayContentContact() {
+	resetContentWrapper();
+	const contentWrapper = document.getElementById('content-wrapper');
+	const contentMenu = document.createElement('div');
+	contentMenu.setAttribute('id', 'content-menu');
+	contentMenu.setAttribute('class', 'tab-content');
+
+	contentMenu.innerHTML = `
+                            <div class="address">
+                              <div class="">
```

Comment:
- [ ] Kindly use es6 arrow function here. Fix this and other affected functions too.

Reply:
Done